codingTracker.jakubmikolajewski/CodingTrackerLibrary/CodingSession.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Globalization;
+
+namespace CodingTrackerLibrary
+{
+    public class CodingSession
+    {
+        private long _id;
```

Comment:
A wonderful thing about modern C# is that you no longer need these backing fields in the current way you are using them (there may be instances where you do, but looking at this class, you don't!).

Your property declaration on line 25 could simply be 
`public long Id { get; }`

All the magic happens in the background and you don't clutter your class up.